src/routes/(auth)/login/+page.svelte

```diff
@@ -68,6 +92,22 @@
 <Container class="flex h-full justify-center items-center">
 	<div class="w-full max-w-screen-md flex flex-col gap-8">
 		<h1>Login</h1>
```

Comment:
Could you please center the 'Login' header and the 'cancel' and 'login' buttons at the bottom. Can you also put the 'Dont have an account', 'forgot password' side by side and center them also.